pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
```

Comment:
What is this file used for since we also have the setup.py ?

Reply:
https://packaging.python.org/en/latest/discussions/setup-py-deprecated/#is-pyproject-toml-mandatory

"While it is not technically necessary yet, it is STRONGLY RECOMMENDED for a project to have a pyproject.toml file at the root of its source tree with a content like this:"